console/src/permissions/SetLayout.tsx

```diff
@@ -0,0 +1,179 @@
+      );
+      const userSpecificPolicies = policies.filter(
+        (p) => p.subjects.length === 1 && p.subjects[0].key === user_.key,
+      );
+      const keys = convertPoliciesToKeys(userSpecificPolicies);
+      const permissions = convertKeysToPermissions(keys);
+      return {
+        ...permissions,
+        keys,
+      };
+    },
+    applyChanges: async ({ client, values, path, prev }) => {
+      if (path === "") return;
+      const policy = path as ConsolePolicy;
+      const previouslyActive = prev as boolean;
+      if (previouslyActive) {
+        const key = values.keys[policy];
+        if (key == null) return;
+        await client.access.policy.delete(key);
+        return;
+      }
+      const newPolicy = await client.access.policy.create({
+        subjects: {
+          type: user.ONTOLOGY_TYPE,
+          key: user_.key,
+        },
+        ...consolePolicyRecord[policy],
+      });
+      values.keys[policy] = newPolicy.key;
+    },
+  });
+
+  const isRootUser = user_.rootUser;
+  if (isRootUser) {
+    addStatus({
+      variant: "error",
+      message: "Root user permissions cannot be modified",
+    });
+    onClose();
+    return <></>;
+  }
+
+  return (
+    <Align.Space style={{ paddingTop: "2rem", height: "100%" }} grow empty>
+      <Align.Space
+        className="console-form"
+        justify="center"
+        style={{ padding: "3rem" }}
+        grow
+      >
+        <Form.Form {...methods}>
+          <Align.Space direction="y">
+            <Text.Text level="h2">Permissions for {user_.username}</Text.Text>
+            {Array.from(consolePolicySet).map((policy, index) => (
+              <Fragment key={index}>
+                {index > 0 && <Divider.Divider direction="x" />}
+                <Align.Space direction="x" key={policy} justify="center">
+                  <Form.SwitchField path={policy} label={foo[policy].label} />
+                  <Text.Text level="p">{foo[policy].description}</Text.Text>
+                </Align.Space>
+              </Fragment>
+            ))}
+          </Align.Space>
+        </Form.Form>
+      </Align.Space>
+      <Nav.Bar location="bottom" size="7.5rem">
+        <Nav.Bar.Start style={{ paddingLeft: "2rem" }}>
+          <Triggers.Text level="small" trigger={SAVE_TRIGGER} />
+          <Text.Text level="small">To Close</Text.Text>
+        </Nav.Bar.Start>
+        <Nav.Bar.End style={{ paddingRight: "2rem" }}>
+          <Button.Button onClick={onClose} triggers={[SAVE_TRIGGER]}>
+            Close
+          </Button.Button>
+        </Nav.Bar.End>
+      </Nav.Bar>
+    </Align.Space>
+  );
+};
+
+type FormItems = {
+  [P in ConsolePolicy]: {
+    label: string;
+    description: string;
+  };
+};
+
+const foo: FormItems = {
+  admin: {
+    label: "Admin",
+    description:
+      "Admin permissions allow the user to manage other users, including registering users and setting permissions for those users.",
+  },
+  schematic: {
+    label: "Schematic",
```

Comment:
The name for this permission is strange. The title makes it seem like it controls everything to do with schematics, but it's just for edit permissions? Why can't we call the permission "Edit Schematics?"

Reply:
Or, even better, whey don't we put this into a separate group called "Schematics", and then we have a toggle called 'edit' with this description. Much easier to understand.